ValheimPlus/GameClasses/CraftingStation.cs

```diff
@@ -12,7 +12,7 @@ class CraftingStationChanges
         [HarmonyPatch(typeof(CraftingStation), "Start")]
         public static class WorkbenchRangeIncrease
         {
-            private static void Prefix(ref float ___m_rangeBuild, GameObject ___m_areaMarker)
+            private static void Prefix(CraftingStation __instance, ref float ___m_rangeBuild, GameObject ___m_areaMarker)
```

Comment:
You need to use "ref CraftingStation __instance"

Reply:
The ref keyword is not needed here, and in fact it's not needed in several other places it is currently being used. Classes in C# are always passed by reference. Adding ref makes it a reference to the reference, which would only be useful if you wanted to replace the caller's representation of the instance itself.

Reply:
But, to be clear, it is still needed on primitive-typed arguments.

Reply:
In C++syntax, it would effectively be the difference between these:
`void Prefix(CraftingStation** __instance)` (with ref)
`void Prefix(CraftingStation* __instance)` (without ref)
but in C#, there's obviously no `->` operator, so the `.` operator always dereferences all pointers.

Needing the ref keyword on classes is a common misconception in C#, and in my opinion the language needs to be altered to be more explicit.